deploy/roles/r-core/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Ensure umask is correct
+  lineinfile:
+    dest: "~/.bashrc"
+    line: umask 0022
+    regexp: (.*)umask(.*)
+    state: present
+
+- name: Install R-core
+  yum:
+    name: R-core
+    state: latest
```

Comment:
This should be `state: "{{ r_core_state }}"` to allow us to control when R is updated.